aidial_adapter_openai/gpt.py

```diff
@@ -26,6 +46,7 @@ async def gpt_chat_completion(
     tokenizer: Tokenizer,
 ):
     discarded_messages = None
+    used_prompt_tokens = None
```

Comment:
Maybe simply `prompt_tokens`.
When you add an adjective to a name of a variable, it implies that there are other kinds of this thing.
That is, `used_prompt_tokens` implies the existence of unused or eastimated prompt tokens.

Reply:
fixed